genbindings.sh

```diff
@@ -64,12 +65,14 @@ fi
 
 rm genbindings_path_map_test_file.c
 
-ENV_TARGET=$(rustc --version --verbose | grep host | awk '{ print $2 }' | sed 's/-/_/g')
 case "$ENV_TARGET" in
 	"x86_64"*)
 		export RUSTFLAGS="$BASE_RUSTFLAGS -C target-cpu=sandybridge"
 		export CFLAGS_$ENV_TARGET="$BASE_HOST_CFLAGS -march=sandybridge -mcpu=sandybridge -mtune=sandybridge"
 		;;
+	"aarch64-apple"*)
```

Comment:
Should probably be `aarch64_apple_darwin`?
```suggestion
	"aarch64_apple_darwin"*)
```

Reply:
Yeah, but it does matter since you re-write `$ENV_TARGET` with `aarch64_apple_darwin` up top, i.e., `-` vs. `_`.